submissions/dafen173/pop-up2/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>PopUp menu</title>
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="style.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+	<body>
+		<nav>
+			<div class="menu">
+				<div class="menuItem" tabindex="1">Mail</div>
+				<div class="menuItem" tabindex="2">Photos</div>
+				
+
+				<input type="checkbox" id="ButtonPopUp" class="popUp" tabindex="3"/>
```

Comment:
all single tags should be without slashes - according to HTML5 specification. Slashes for single tags uses only for HTML4 or lower versions.
Could you please fix it?